Assignment2/AlvaroMK-Alvaro Ajenjo Jurado-fromRDFtoJSON-LD.jsonld.txt

```diff
@@ -0,0 +1,37 @@
+{
+	"@context":{
+		"base": "http://example.org/",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+	},
+	"@graph":[
+		{
+			"@id":"base:Class01",
+			"base:includes":{
+				"@id":"base:Sensor029",
+				"base:hasMeasurement":{
+					"@id":"base:Measurement8401",
+					"base:hasTemperature":{
+						"@type": "xsd:int",
+						"@value":29,
+					},
+					"base:atTime":{
+						"@value": "2010-06-12T12:00:12",
+						"@type":"xsd:dateTime",
+					}
+				}
+			}
+
+		},
+
+		"Computer101":{
+			"@id":"base:Class01",
+			"base:hasOwner":{
+				"@id":"base:User10A",
+				"base:hasName":{
+					"@value": "Pedro",
+					"@type": "xsd:string",
+				}
+			}
+		}
+	}
+}

```

Comment:
This file has errors. The property "base:includes" must relate Sensor029 and Computer101. 
In addition, change the id of Computer101 since it isn't a Class01.